### PR TITLE
[IA-2341] Add stop_start action to kubernetes-app resources

### PR DESCRIPTION
**Pull request by rtitle**
Ticket: https://broadworkbench.atlassian.net/browse/IA-2341

This will support Leo changes (not quite ready yet) to stop, start, and auto-stop Kubernetes (e.g. Galaxy) applications.

Planning to test end-to-end before merging this.

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
I tested this with Leo, I think it's all working. Requesting review (@dvoet @gpolumbo-broad)

**Comment by rtitle** [comment]
Seems unrelated test failure, jenkins retest

**Comment by rtitle** [comment]
now a fiab-start failure :( jenkins retest

**Comment by rtitle** [comment]
think it's sorted now, jenkins retest

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -451,14 +451,17 @@ resourceTypes = {
       status = {
         description = "view details and configuration of the kubernetes application"
       }
+      stop_start = {
```
Comment (dvoet) [inline src/main/resources/reference.conf]: why not 2 actions: start and stop?

Comment (rtitle) [reply src/main/resources/reference.conf]: I didn't think there was a situation where one should be able to stop and not start an app (or vice versa). Runtimes work similarly: https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L399

(We grant the "manager" role the ability to delete, but not stop/start other users' apps.)

Comment (rtitle) [reply src/main/resources/reference.conf]: I could make it 2 actions if that's preferred though

Comment (rtitle) [reply src/main/resources/reference.conf]: I made it 2 actions, probably makes more sense

**Comment by rtitle** [comment]
@dvoet circling back, can I have a thumb on this? I hope to merge the Leo side soon, but this needs to go first. (Thus far I’ve been testing on a fiab with both Leo and Sam branches deployed.) Thanks!

**Review by Qi77Qi — APPROVED** [review]

**Review by dvoet — APPROVED** [review]

File: src/main/resources/reference.conf (hunk 1 above)
Comment (dvoet) [reply src/main/resources/reference.conf]: never know what the future holds :)

**Pull request merged**